packages/server/lib/html/multi-domain-iframe.html

```diff
@@ -7,6 +7,7 @@
   <body>
     <script type="text/javascript">
       document.domain = '{{domain}}';
```

Comment:
the browser converts document.domain to punycode by default, how helpful, but doesn't provide any API's to convert it back to unicode. So I attach the domain to window below.